Detectors/FIT/FV0/simulation/src/Detector.cxx

```diff
@@ -131,10 +131,10 @@ Bool_t Detector::ProcessHits(FairVolume* v)
   return kTRUE;
 }
 
-o2::v0::Hit* Detector::addHit(Int_t trackId, Int_t cellId,
-                              TVector3 startPos, TVector3 endPos,
-                              TVector3 startMom, double startE,
-                              double endTime, double eLoss, Int_t particlePdg)
+o2::fv0::Hit* Detector::addHit(Int_t trackId, Int_t cellId,
+                               TVector3 startPos, TVector3 endPos,
```

Comment:
No need to use TObject based virtual object to transfer the position/momentum, please change this to ``Point3D<float>`` or ``Vector3D<float>``, also in related methods

Reply:
Fixed now